statistics/src/main/java/com/duckduckgo/app/statistics/api/featureusage/FeatureSegmentsPixelSender.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2023 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.statistics.api.featureusage
+
+import com.duckduckgo.app.statistics.api.RefreshRetentionAtbPlugin
+import com.duckduckgo.browser.api.UserBrowserProperties
+import com.duckduckgo.di.scopes.AppScope
+import com.squareup.anvil.annotations.ContributesMultibinding
+import javax.inject.Inject
+
+@ContributesMultibinding(AppScope::class)
+class FeatureSegmentsPixelSender @Inject constructor(
```

Comment:
Name is kinda weird, this implements `RefreshRetentionAtbPlugin` which is not related with pixels at all. Perhaps something like `SegmentsRefreshRetentionAtbPlugin` ?

Reply:
I followed the same pattern we have for `FeatureRetentionPixelSender` and `DeviceShieldRetentionPixelSender` which have similar function and implement the same interface so I would recommend to leave this name to have consistency